hack/verify-sh.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+find . -type f -name '*_test.sh' -exec {} \;
```

Comment:
```
# ./yes.sh ; echo $?
0
# ./no.sh ; echo $?
1
# find . -name '*sh' -exec '{}' \; ; echo $?
0
# gfind . -name '*sh' -exec '{}' \; ; echo $?
0
```

My `find`s don't return an error if one of the `-exec`s fails.

Reply:
Good catch. It should be okay now. I also changed the scripts to test-... I was thinking about this as well and have no real preference if they are verify- or test-, for now.